app/models/miq_server.rb

```diff
@@ -487,6 +487,19 @@ def self.zone_is_modifiable?
     Zone.visible.in_my_region.count > 1
   end
 
+  def self.audit_vm_totals
+    total_vms     = 0
+    total_hosts   = 0
+
+    ExtManagementSystem.all.each do |e|
+      vms     = e.all_vms_and_templates.count
+      hosts   = e.all_hosts.count
+      total_vms += vms
+      total_hosts += hosts
+    end
+    $audit_log.info("Under Management: VMs: [#{total_vms}], Hosts: [#{total_hosts}]")
```

Comment:
For parsing purposes, let's change the log message to be a JSON output.  Something like

```ruby
totals = {"vms" => total_vms, "hosts" => total_hosts}
$audit_log.info("Under Management: #{totals.to_json}")
```

This way, your parser can just parse simple JSON, and if we ever need to add stuff in the future we can and it won't break your parser.

Reply:
OK, I've changed it. Output is:
`[----] I, [2020-02-14T11:06:26.844294 #84536:3fde43432e3c]  INFO -- : Q-task_id([audit_managed_resources]) Under Management: {"vms":0,"hosts":0}`